content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -260,7 +260,9 @@ unifiedControlPlaneImage: ""
 ### Adding kube-proxy parameters {#kube-proxy}
 
 For information about kube-proxy parameters in the MasterConfiguration see:
-- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/kubeproxyconfig/v1alpha1#KubeProxyConfiguration)
+- [kube-proxy](https://godoc.org/k8s.io/kubernetes/pkg/proxy/apis/config#KubeProxyConfiguration)
+For information about enabling IPVS mode with kubeadm see:
```

Comment:
@riverzhang try adding an empty line above this.

Reply:
Thanks，it is ok when add empty line.